disco_aws_automation/resource_helper.py

```diff
@@ -211,3 +204,17 @@ def check_written_s3(object_name, expected_written_length, written_length):
         raise S3WritingError(
             "{0} is not written correctly to S3 bucket".format(object_name)
         )
+
+
+def backoff(cycle):
+    """
+    This function takes as input an integer that represents a cycle count,
+    calculates jitter and executes sleep for the calculated time.
+    The value of 'cycle' must be an integer greater than 0.
+    """
+    base = 3
+    if cycle is None or cycle < 1:
+        raise ValueError('Value of cycle must be int > 0')
+
+    time.sleep(min(MAX_POLL_INTERVAL, randint(base, cycle * 3)))
+    return cycle + 1
```

Comment:
That would not be a very good indicator of the time passed, as STATE_POLL_INTERVAL appears to be hard coded as `2` but the backoff function is very much not hardcoded to `2`. Just adding that worked before because STATE_POLL_INTERVAL was how long things actual slept.